examples/tutorials/analysis-1d/parameter_ul.py

```diff
@@ -0,0 +1,218 @@
+
+
+######################################################################
+# Check setup
+# -----------
+#
+
+from gammapy.utils.check import check_tutorials_setup
+
+check_tutorials_setup()
+
+######################################################################
+# Load observation
+# ----------------
+#
+# We will now use a precomputed blazar dataset to see how to contrain
+# model parameters. Detailed modeling of this dataset may be found in the
+# :doc:`/tutorials/analysis-1d/ebl` notebook. We will try to
+# constrain the spectral cutoff in the source. To see
+#
+
+dataset_onoff = SpectrumDatasetOnOff.read(
+    "$GAMMAPY_DATA/PKS2155-steady/pks2155-304_steady.fits.gz"
+)
+dataset_onoff.peek()
+plt.show()
+
+
+######################################################################
+# Fit spectrum
+# ------------
+#
+# Fit a spectral model with a cutoff
+#
+
+spectral1 = ExpCutoffPowerLawSpectralModel()
+spectral1.amplitude.value = 5e-12
+spectral1.alpha.value = 2.0
+
+model_pks = SkyModel(spectral1, name="model_pks")
+dataset_onoff.models = model_pks
+
+fit = Fit()
+res_pks = fit.run(dataset_onoff)
+print(res_pks.models)
+
+
+######################################################################
+# We see that the parameter `lambda_` is not well constrained. In this
+# case, it can be helpful to look at the likelihood profile of the parameter.
+# We will use the `~gammapy.modeling.Fit.stat_profile` function, and refit the other free
+# parameters in the process by setting `reoptimize=True`
+#
+
+parameter = model_pks.parameters["lambda_"]
+parameter.scan_n_values = 25
+parameter.scan_min = 0.05
+parameter.scan_max = 5
+parameter.interp = "log"
+profile = fit.stat_profile(datasets=dataset_onoff, parameter=parameter, reoptimize=True)
+
+
+######################################################################
+# `profile` is a dictionary that stores the values of `lambda_` and the
+# corresponding likelihood values. Let's try to visualise it.
+#
+
+values = profile["model_pks.spectral.lambda__scan"]
+loglike = profile["stat_scan"]
+
+ax = plt.gca()
+ax.plot(values, loglike - np.min(loglike))
+ax.set_xlabel("Cutoff value (1/TeV)")
+ax.set_ylabel(r"$\Delta$TS")
+secay = ax.secondary_yaxis("right", functions=(ts_to_sigma, sigma_to_ts))
+secay.set_ylabel("Significance [$\sigma$]")
+plt.title("Cutoff likelihood profile", fontsize=20, y=1.05)
+plt.show()
+
+
+######################################################################
+# Constrain the limits on the parameter
+# -------------------------------------
+# Thus, this dataset does not yield a significant spectral cutoff in
+# PKS2155-304. In particular, we cannot constrain the lower limit from
+# this profile. We can compute the limits of the cutoff from the Probability Density Function (PDF).
+# For this, we will first convert the likelihood profile into a normalised PDF with `likelihood_to_pdf`
+# and then use it to compute the intervals from the Cumulative Distribution Function (CDF).
```

Comment:
This whole section is not correct. The exponential of the likelihood profile does not yield a pdf, because likelihoods are not probabilities.
Technically, the profile likelihood is not even a likelihood but usually behaves like one (hence the validity of measuring confidence intervals the way we do). See:
- https://myweb.uiowa.edu/pbreheny/7110/f20/notes/11-04.pdf
- https://arxiv.org/pdf/1903.00162

In general, we should remember that profile likelihoods can be biased. They don't necessarily give the correct minimum when there are many nuisance parameters and small statistics. So finding the best fit value from the profile likelihood is not the first we should show.